client/my-sites/subscribers/components/add-subscribers-modal/style.scss

```diff
@@ -67,11 +67,6 @@
 		}
 	}
 
-	.add-subscriber__form-submit-btn {
```

Comment:
This float was taking the button out of the page flow so that it hit the bottom edge of the modal. I can add a different style rule if it needs to be on the right.